### ISSUE TITLE
Your Logo is unreadable in DarkTheme Github

### ISSUE THREAD

**Issue by conrad-lippert-zajaczkowski-8451** [seata/seata#5919]
The logo looked like just blue lines on the dark theme of Github. I couldn't read it until I figured that out.

**Comment by slievrly** [comment]
Yes, there is a problem. Seata has 2 sets of logos, one in black and blue, and one in white and gray. These two sets of logos will have similar problems when matching github themes with similar colors.
